.github/workflows/forward-compatibility.yaml

```diff
@@ -0,0 +1,131 @@
+# Copyright NVIDIA CORPORATION
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+name: Forward Compatibility
+
+on:
+  schedule:
+    - cron: '0 2 * * 1'  # Weekly on Monday at 2 AM UTC
+  workflow_dispatch:
+    inputs:
+      toolkit_image:
+        description: 'Override container-toolkit image'
+        required: false
+        type: string
+      device_plugin_image:
+        description: 'Override device-plugin image'
+        required: false
+        type: string
+      mig_manager_image:
+        description: 'Override mig-manager image'
+        required: false
+        type: string
```

Comment:
Why do these variables need to be inputs? Aren't we always resolving the images in this workflow itself?

Reply:
removed